src/main/java/TaskList.java

```diff
@@ -0,0 +1,154 @@
+import exceptions.EmptyNameException;
+import exceptions.InvalidTaskIndexException;
+import exceptions.NoTasksException;
+import objects.Deadline;
+import objects.Event;
+import objects.Task;
+import objects.Todo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private static List<Task> tasks = new ArrayList<>();
+
+    public List<Task> getTasks() {
+        return tasks;
+    }
+
+    /**
+     * Creates a new objects.Todo object and adds it to the tasks list.
+     * @param inputs array of input strings (e.g. ["todo", "wash dishes"]
+     */
+    public void addTodo(String[] inputs) throws EmptyNameException {
+        if (inputs.length == 1) {
+            throw new EmptyNameException("Todo name cannot be empty...");
+        }
+        StringBuilder todoName = new StringBuilder();
+        for (int i = 1; i < inputs.length - 1; i++) {
+            todoName.append(inputs[i]).append(" ");
+        }
+        // Edge case: When inputs.length is 2, the todoName is not captured by the loop above.
+        todoName.append(inputs[inputs.length - 1]);
```

Comment:
To me, this section is a little confusing (especially for the other tasks). Perhaps include some more explanation, or alternatively, you can consider using regular expressions or just using `String::split` with a limit or taking a subarray from inputs and then joining them with `String::join`.